### PR TITLE
Feature/vtx 1751/add company

### PR DESCRIPTION
**Pull request by StrDragos**
[VTX-1751]

Add company id to the payload so that we can remove authorization from the implementation. 

[VTX-1751]: https://coralogix.atlassian.net/browse/VTX-1751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: com/coralogix/integrations/grpc/external/v1/ExternalIntegrationConfigService.proto [hunk 1]
```diff
@@ -60,6 +63,7 @@ message CreateExternalIntegrationConfigResponse {
 
 message ReplaceTokenRequest {
   google.protobuf.StringValue id = 1;
+  google.protobuf.Int32Value companyId = 2;
```
Comment (xBATx) [inline com/coralogix/integrations/grpc/external/v1/ExternalIntegrationConfigService.proto:66]: Do we need `companyId` also for other endpoints than the `create`? I think we shouldn't at least for this one and the delete.
I don't have many insights about the domain but considering `ReplaceTokenRequest` is the name, shouldn't it also contain the `token`? :D

**Review by xBATx — APPROVED** [review]
LGTM 👍

File: com/coralogix/integrations/grpc/external/v1/ExternalIntegrationConfigService.proto (hunk 1 above)
Comment (stsatlantis) [reply com/coralogix/integrations/grpc/external/v1/ExternalIntegrationConfigService.proto:66]: based on the id it just creates a new one, using the information already stored in the db :)

Comment (StrDragos) [reply com/coralogix/integrations/grpc/external/v1/ExternalIntegrationConfigService.proto:66]: I have add it on all endpoints that are using company id in the implementation.

**Review by kolov — APPROVED** [review]

**Pull request merged**